telco-hub/install/agent-config.yaml

```diff
@@ -0,0 +1,162 @@
+apiVersion: v1alpha1
+kind: AgentConfig
+metadata:
+  name: myhub # need to match the same name put in install-config
+rendezvousIP: 192.168.56.25 # one of the master IP 
+hosts:
+
+  - hostname: master-0
+    role: master
+    interfaces:
+      - macAddress: 94:6d:ae:db:0b:64
+        name: eno12399np0
+      - macAddress: 94:6d:ae:db:0b:65
+        name: eno12409np1
+    networkConfig:
+      interfaces:
+      - name: infra-bond.46
+        type: vlan
+        state: up
+        vlan:
+          base-iface: infra-bond
+          id: 46
```

Comment:
Control plane node 1 is on vlan 46 but the other two are on vlan 56. Is that intentional? I would expect all 3 to be on the same vlan.